src/applications/personalization/rated-disabilities/components/TotalRatingStates.jsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
 import PercentageCalloutBox from '../components/PercentageCalloutBox';
+import recordEvent from 'platform/monitoring/record-event';
```

Comment:
nit: I prefer putting `platform` imports above relative imports but below imports from `node_modules` but that isn't a rule we have.